lighthouse-cli/cli-flags.js

```diff
@@ -70,7 +70,8 @@ function getFlags(manualArgv) {
         'blocked-url-patterns': 'Block any network requests to the specified URL patterns',
         'disable-storage-reset':
             'Disable clearing the browser cache and other storage APIs before a run',
-        'disable-device-emulation': 'Disable Nexus 5X emulation',
+        'disable-device-emulation': 'Disable all device form factor emulation',
+        'emulated-form-factor': 'Controls the emulated device form factor (mobile vs. desktop)',
```

Comment:
Should we also mention the flag precedence here, like "Controls the emulated device form factor (mobile vs. desktop) if emulation not disabled" or something. It gets weird that `none` *also* disables it, but it isn't too confusing...

(could also add something about `Deprecated: use '--emulated-form-factor none'.` to the `disable-device-emulation` description)

Reply:
should also update the readme with new `--help` output

Reply:
yeah good call, done